tests/models/blaise/test_blaise_case_information_model.py

```diff
@@ -33,6 +33,7 @@ def valid_case_data_dictionary() -> Dict:
         "catiMana.CatiCall.RegsCalls[1].DialResult": "1",
         "qRotate.RDMktnIND": "2",
         "qRotate.RHOut": "310",
+        "qDataBag.TLA": "LMB",
```

Comment:
If we are not reading TLA from qDataBag do we still need this line in this test?

Reply:
I purposely added this and updated the test to assert it sets the TLA from the questionnaire name and not the qDataBag. 👍